main/convert.js

```diff
@@ -108,7 +112,19 @@ const mute = PCancelable.fn(async (inputPath, onCancel) => {
   return mutedPath;
 });
 
-const convert = getConvertFunction(true);
+const convert = getRunFunction(true);
+const compress = (outputPath, options, args) => {
+  options.onProgress(0, 0, 'Compressing');
```

Comment:
```suggestion
  options.onProgress(0, undefined, 'Compressing');
```

Since the second argument is a string, I think `undefined` or `''` would make more sense than 0